src/main/java/bio/terra/service/filedata/google/firestore/FireStoreDependencyDao.java

```diff
@@ -211,7 +214,8 @@ public void removeSnapshotFileDependency(Dataset dataset, String snapshotId, Str
                 return null;
             }
             if (documents.size() > 1) {
-                throw new FileSystemCorruptException("Found more than one document for a file dependency");
+                throw new FileSystemCorruptException(
+                    "Found more than one document for a file dependency - fileId: " + fileId);
```

Comment:
Thank you for adding the file id! Is there an easy way to look up what that is for the user? They just need the dataset id, right?

Reply:
Yes, they would need the dataset id. This kind of corruption error should (when we have everything hooked up)5generate an alert to whoever is on call. We would not expect a user to figure this one out